### ISSUE TITLE
Get a localisation view in NUsight with dots/particles/etc

### ISSUE THREAD

**Issue by LachlanCourt** [NUbots/NUbots#738]
(currently works in NUsight, want a covariance view with the confidence of the localisation view)

**Comment by ysims** [comment]
I don't think this is relevant anymore, but you might want to check if there's any leftover code in NUsight to clean up. @sebastianbrooker2 
Can possibly be closed, but check first.
